pkg/reconciler/v1alpha1/streamingprocessor/streamingprocessor.go

```diff
@@ -330,8 +368,50 @@ func (c *Reconciler) createDeployment(processor *streamingv1alpha1.Processor) (*
 	}
 	return c.KubeClientSet.AppsV1().Deployments(processor.Namespace).Create(deployment)
 }
-
 func deploymentSemanticEquals(desiredDeployment, deployment *appsv1.Deployment) bool {
 	return equality.Semantic.DeepEqual(desiredDeployment.Spec, deployment.Spec) &&
 		equality.Semantic.DeepEqual(desiredDeployment.ObjectMeta.Labels, deployment.ObjectMeta.Labels)
 }
+
+func scaledObjectSemanticEquals(desiredScaledObject, scaledObject *kedav1alpha1.ScaledObject) bool {
```

Comment:
the SemanticEquals method is traditionally after the reconcile method for the type